transcendence-app/Dockerfile

```diff
@@ -25,14 +26,15 @@ ENV NODE_ENV=production
 RUN apk add --no-cache dumb-init \
   && mkdir -p ${TRANSCENDENCE_APP_DATA} \
   && chown -R node:node ${TRANSCENDENCE_APP_DATA}
-WORKDIR  /transcendence-app
+WORKDIR  /service
```

Comment:
why did you call this folder (the monorepo root, I assume) service?

Reply:
Because transcendence-app was converted into a subfolder of the main path and, since these docker images are meant to be doployed as services, I think that was a more less accurate denomination.